src/main/java/Greetings.java

```diff
@@ -0,0 +1,48 @@
+public class Greetings {
+	private static final String LOGO =
+			" ____        _        \n"
+					+ "|  _ \\ _   _| | _____ \n"
+					+ "| | | | | | | |/ / _ \\\n"
+					+ "| |_| | |_| |   <  __/\n"
+					+ "|____/ \\__,_|_|\\_\\___|\n";
+	private static final String OPENING_LINE =
+			"Hello! I'm Duke"
+					+System.lineSeparator()
+					+ "What can I do for you?";
+	private static final String EXIT_LINE =
+			"Bye. Hope to see you again soon!";
+	private static final String DIVIDER =
+			"______________________________";
+	private static final String ERROR = "Hey! Error!";
+
+	public Greetings() {
+	}
+
+	public void printGreetings() {
+		System.out.println("Hello from\n" + LOGO);
+		printDivider();
+	}
+
+	public void printOpeningLine() {
+		printDivider();
+		System.out.println(OPENING_LINE);
+		printDivider();
+	}
+
+	public void printExitLine() {
+		printDivider();
+		System.out.println(EXIT_LINE);
+		printDivider();
+	}
+
+	public void printDivider() {
+		System.lineSeparator();
+		System.out.println(DIVIDER);
+	}
+
+	public void printErrorMessage() {
+		printDivider();
+		System.out.println(ERROR);
+		printDivider();
+	}
+}
```

Comment:
There's some repetition here i.e. printDivider() -> sysout something -> printDivider(). Might be good to extract this pattern out to a method.